pkgs/applications/science/math/scidavis/default.nix

```diff
@@ -0,0 +1,30 @@
+{stdenv, fetchurl, qwt, gsl, qwtPlot3D, muparser, zlib, qt4, qmake4Hook, python2, python2Packages
```

Comment:
please don't pass in python2 as python2Packages already has the attribute python